packages/kotti-ui/source/kotti-filter/KtFilter.vue

```diff
@@ -1,65 +1,46 @@
 <template>
 	<div class="kt-filter">
-		<div class="kt-filter__content">
-			<div
-				v-for="(filter, index) in value"
-				:key="filter.key"
-				class="kt-filter__filter-row"
-			>
-				<FilterRow
-					:column="getColumn(filter.key)"
-					:columnOptions="getColumnOptions(filter.key)"
-					:filter="filter"
-					:index="index"
-					@removeClick="handleRemoveFilter"
-					@selectColumn="handleSelectColumn"
-					@selectOperation="handleSelectOperation"
-					@setValue="handleSetValue"
-				/>
-			</div>
-			<div v-show="isAddingFilter" class="kt-filter__filter-row">
-				<FilterRow
-					:columnOptions="getColumnOptions()"
-					:index="value.length"
-					@removeClick="handleHideAddingFilter"
-					@selectColumn="handleAddFilter"
-				/>
-			</div>
+		<div v-if="hasSearchFilter" class="kt-filter__search-field">
+			<KtFieldText v-model="searchValue" hideValidation size="small" />
 		</div>
-		<div class="kt-filter__footer">
-			<ButtonLink
-				:disabled="isAddingFilter"
-				:icon="Yoco.Icon.PLUS"
-				:label="translations.addFilterLabel"
-				@click="handleShowAddingFilter"
-			/>
-			<ButtonLink
-				:disabled="value.length === 0"
-				:icon="Yoco.Icon.CLOSE"
-				:label="translations.clearAllLabel"
-				@click="handleClear"
+		<div
+			v-on-clickaway="hideFilterList"
+			class="kt-filter__filter-list-container"
+		>
+			<KtButton
+				:label="translations.filtersLabel"
+				type="secondary"
+				@click="toggleFilterListVisibility"
 			/>
+			<div v-show="isFilterListVisible" class="kt-filter__filter-list">
+				<FilterList
+					:columns="getFilterColumns"
+					:filters="value"
+					:isLoading="isLoading"
+					@input="setFilters"
+				/>
+			</div>
 		</div>
 	</div>
 </template>
 
 <script lang="ts">
-import { Yoco } from '@3yourmind/yoco'
+import { KtButton } from '@3yourmind/kotti-ui'
 import { computed, defineComponent, ref } from '@vue/composition-api'
+import { mixin as clickaway } from 'vue-clickaway'
 
 import { useTranslationNamespace } from '../kotti-translation/hooks'
 import { Kotti } from '../types'
 
-import ButtonLink from './components/ButtonLink.vue'
-import FilterRow from './components/FilterRow.vue'
-import { getFilterWithInitialState } from './utils'
+import FilterList from './components/FilterList.vue'
 
 export default defineComponent<Kotti.Filter.InternalProps>({
 	name: 'KtFilter',
 	components: {
-		ButtonLink,
-		FilterRow,
+		FilterList,
+		KtButton,
 	},
+	mixins: [clickaway],
```

Comment:
Why is this necessary? We really don’t want to use `mixins` anymore

Reply:
Is there an alternative to this? It is here due to design, the filter list is supposed to be shown in a dropdown menu like container when clicking a button.

Reply:
Yes, there’s `@3yourmind/vue-use-tippy`